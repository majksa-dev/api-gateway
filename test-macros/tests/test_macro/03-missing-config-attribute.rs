#[helper::test(servers = 1)]
fn single_server(ctx: helper::Context) {}

fn main() {}
