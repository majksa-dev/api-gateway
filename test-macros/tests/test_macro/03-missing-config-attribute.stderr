error: config attribute not found
 --> tests/test_macro/03-missing-config-attribute.rs:1:1
  |
1 | #[helper::test(servers = 1)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `helper::test` (in Nightly builds, run with -Z macro-backtrace for more info)
